### PR TITLE
Add dtstart parameter, remove default

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: todoman/cli.py [hunk 1]
```diff
@@ -52,13 +52,13 @@ def _todo_property_options(command):
         help=('The due date of the task, in the format specified in the '
               'configuration file.'))(command)
     click.option(
-        '--start', '-s', default='', callback=_validate_date_param,
+        '--dtstart', '-s', default='', callback=_validate_date_param,
```
Comment (hobarrera) [inline todoman/cli.py]: I'd rather call this `start`. Even if vCard field name is `dtstart`, I think that for most users, start is more understandable (we *understand* vCard stuff, but most don't).

Comment (untitaker) [inline todoman/cli.py]: I thought similarly, but dtstart (the Todo property) is exposed through the `--sort` option, so we would have to rename that as well.

Comment (hobarrera) [inline todoman/cli.py]: I'd much rather expose `start` than `dtstart` (including in `sort`).

**Comment by untitaker** [comment]
Renamed `Todo.dtstart` to `Todo.start`.

**Comment by hobarrera** [comment]
LGTM. Feel free to merge this in! Thanks!

**Comment by untitaker** [comment]
:100:

**Pull request merged**